plugin/runner/runner.go

```diff
@@ -0,0 +1,34 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package runner
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/utils/ulimit"
+	"github.com/ava-labs/avalanchego/version"
+	"github.com/ava-labs/avalanchego/vms/rpcchainvm"
+
+	"github.com/ava-labs/subnet-evm/plugin/evm"
+)
+
+func Run() {
```

Comment:
should we add a string argument so the external repo can add some text to the PrintVersion message?

Reply:
Agreed it would be nice to allow an external repo to optionally specify some additional output to be included in `PrintVersion()`.

Reply:
added